test/modules/plugins/erc4337/SafeProtocol4337Module.spec.ts

```diff
@@ -0,0 +1,84 @@
+import EntryPoint from "@account-abstraction/contracts/artifacts/EntryPoint.json";
+import { Signer } from "ethers";
+import { ethers, deployments } from "hardhat";
+
+import { MODULE_TYPE_PLUGIN, MODULE_TYPE_FUNCTION_HANDLER, PLUGIN_PERMISSION_EXECUTE_CALL } from "../../../../src/utils/constants";
+
+describe("SafeProtocol4337Module", async () => {
+    const setupTests = deployments.createFixture(async ({ deployments }) => {
+        await deployments.fixture();
+        const [deployer, owner, user, bundler] = await ethers.getSigners();
+
+        const registry = await ethers.deployContract("SafeProtocolRegistry", [owner.address]);
+        const manager = await ethers.deployContract("SafeProtocolManager", [owner.address, await registry.getAddress()]);
+
+        const entrypoint = await deployEntryPoint(deployer);
+        const module = await ethers.deployContract("SafeProtocol4337Module", [await entrypoint.getAddress()]);
+        const handler = await ethers.getContractAt("ISafeProtocol4337Handler", await module.getAddress());
+
+        const account = await ethers.deployContract("TestExecutor", [await manager.getAddress()]);
```

Comment:
This test is more of a unit test, right? because the correctness of storage access will not be checked

Reply:
Exactly, there is a `user-operation` script that can be used to verify correctness of storage access.